### PR TITLE
Create jenkins.md

### PR DESCRIPTION
**Pull request by hfossli**

### CODE REVIEW

**Comment by hfossli** [comment]
Forhåndsvisning https://github.com/agensdev/playbook-ios/blob/8ecb4b1d6c4b78e14c05135d81d4153eecae7129/jenkins.md

**Comment by kostiakoval** [comment]
## The playbook is still ALIVE!!! 😍

**Comment by hfossli** [comment]
Haha! Yes! 😂 It is a good format for discussion and documentation IMO

File: jenkins.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## Useful functions in fastlane
+
+Using `workout` means you don't need to rebuild project when working and building locally. 
+It also means you don't need to change provisioning profiles etc in xcode project making development easier.
+
+```rb
+  lane :build do
+    workout(
+      workspace: 'NRKRadio.xcworkspace',
```
Comment (langtind) [inline jenkins.md]: Kundenavn må fjernes

File: jenkins.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      configuration: 'Release',
+      method: 'enterprise',
+      team_id: 'US7P5RSD3X',
```
Comment (langtind) [inline jenkins.md]: Team id må fjernes

File: jenkins.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      scheme: 'DRadio',
+      provisioningProfiles: {
+        "no.agens.enterprise.DRadio" => "match InHouse no.agens.enterprise.DRadio"
```
Comment (langtind) [inline jenkins.md]: Kundenavn må fjernes

**Pull request merged**